template/ppx/template/test/{{ project_snake }}_test.mli

```diff
@@ -0,0 +1,3 @@
+open Ppxlib
```

Comment:
Shouldn't this file be `.ml` and not `.mli`? Also should we add a dune file with a test stanza for it?

Reply:
You're right.

There is a story to this. The original template had no `test/{{ project_snake }}_test.mli` and `make test` was complaining that the `mli` file is missing. I was confused and tried to copy the empty `bin/main.mli` to `test/{{ project_snake }}_test.mli`. Missed an `i` obviously. But how can it first ask for an `mli` and then accept an invalid `mli`?

I'll try again.